templates/publisher/publicise/github_badges.html

```diff
@@ -1,5 +1,37 @@
 {% extends 'publisher/publicise/publicise_layout.html' %}
 
 {% block publicise_content %}
-  <h4>Github badges page</h4>
+  <h4>Promote your charm using an embeddable GitHub badge</h4>
+  <div class="row">
+    <div class="col-2"></div>
+    <div class="col-6">
+      <p>
+        <a href="/{{ package.name }}">
+          <img alt="{{ package.title }}" src="/{{ package.name }}/badge.svg" />
```

Comment:
It's not useful for the alt to just say a title. It should just be empty in this case

Reply:
ok